static/skywire-manager-src/src/app/components/vpn/pages/vpn-server-list/add-vpn-server/add-vpn-server.component.ts

```diff
@@ -0,0 +1,102 @@
+import { Component, ViewChild, ElementRef, Inject, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skywire/pullRequest?prid=7163595)
```suggestion
import { FormBuilder, FormGroup, Validators } from "@angular/forms";
```